server/server.go

```diff
@@ -0,0 +1,165 @@
+package server
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/merkletree"
+	"github.com/coniks-sys/coniks-go/crypto"
+)
+
+const (
+	Capacity = 1000000
+)
+
+type ServerConfig struct {
+	SigningKeyPath      string  `toml:"signing_key"`
+	Address             string  `toml:"address"` // address:port
+	LoadedHistoryLength int64   `toml:"loaded_history_length"`
+	PoliciesPath        string  `toml:"policies"`
+	TLS                 tlsInfo `toml:"tls"`
+}
+
+type tlsInfo struct {
+	TLSCertPath string `toml:"cert"`
+	TLSKeyPath  string `toml:"key"`
+}
+
+type serverState struct {
+	sync.RWMutex
+	pad *merkletree.PAD
```

Comment:
If we go with the changes suggested in https://github.com/coniks-sys/coniks-go/pull/29/commits/e731ef9812bc9484dbaaef62a4d92d24de8250f4#r71962278, this could be ``directory *ConiksDirectory``